script.js

```diff
@@ -20,7 +20,6 @@ function enableScroll() {
   window.onscroll = '';
 }
 
-// Work Section 1
 const cards = [
   {
```

Comment:
- [ ] Great job, the only issues I have found have to do with the layout of items in both the desktop and mobile view.

#### Desktop

Your design
![image](https://user-images.githubusercontent.com/81932478/189478822-2c1c4acb-d58b-4eca-b70e-723a06377d19.png)

Figma design
![image](https://user-images.githubusercontent.com/81932478/189478956-3b6f61c4-f1f5-4796-be81-e7591f6e4794.png)

- The text and should be aligned to where the image starts on the left
- There sould be a margin on the right from the corner of the `see live` buttons and `languages`.
-  The `see sources` button should be aligned with the image above as it is in the given design.

#### Mobile 

Your design
![image](https://user-images.githubusercontent.com/81932478/189479241-b632acc2-4613-4c2d-be58-20fa9ffd68f9.png)

Figma design
![image](https://user-images.githubusercontent.com/81932478/189479324-ffed18f8-1ddd-4e36-b720-ccbcc5bde369.png)


- The mobile view does not have the transparent background, kindly add one
- Kindly align your content to the middle so that the margin on the right and on the left is equal as in the Figma design.
- Kindly add a side scroll so that the bottom content such as the buttons can be seen in your design
-